src/init.lua

```diff
@@ -3,14 +3,19 @@
 ]]
 
 local GlobalConfig = require(script.GlobalConfig)
-local createReconciler = require(script.createReconciler)
 local createReconcilerCompat = require(script.createReconcilerCompat)
-local RobloxRenderer = require(script.RobloxRenderer)
 local strict = require(script.strict)
 local Binding = require(script.Binding)
+local VirtualTree = require(script.VirtualTree)
 
-local robloxReconciler = createReconciler(RobloxRenderer)
-local reconcilerCompat = createReconcilerCompat(robloxReconciler)
+local function mount(element, hostParent, hostKey)
+	return VirtualTree.mount(element, {
```

Comment:
should VirtualTree.mount have the signature `mount(element, hostParent, hostKey)` so we don't have to declare a local function here?

Reply:
I wanted `VirtualTree.mout` to accept a reconciler so it can be changed, but I did not wanted to have a 4 arguments function with three optional ones. It makes weird call when you want to pass only the reconciler (`VirtualTree.mount(element, nil, nil, reconciler)`). If one day mount needs a new argument (if we make a true shallow renderer for example), it's going to get worse.

So I decided to use a table to make it easier in the future if we add/remove other optional arguments.